Java/measureeval/src/main/java/com/lantanagroup/link/measureeval/utils/CqlUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.lantanagroup.link.measureeval.utils;
+
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.Library;
+import org.hl7.fhir.r4.model.ResourceType;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Optional;
+
+public class CqlUtils {
+    public static String getCql(Bundle bundle, String libraryId, String range) {
```

Comment:
Do we want to throw `ResponseStatusException` from this method?  It's appropriate when called from the new `$cql` endpoint, but not from `CqlLogAppender` (which would bring down a measure evaluation, maybe halfway through handling a Kafka event).  In the latter case, we'd probably want to log a warning or just fail silently.

Reply:
Changed to `NotFoundException` in `CqlUtils` and handled that exception in its usages.